test/test_prototype_transforms_functional.py

```diff
@@ -1060,3 +1061,13 @@ def test_equalize_image_tensor_edge_cases():
     inpt[..., 100:, 100:] = 1
     output = F.equalize_image_tensor(inpt)
     assert output.unique().tolist() == [0, 255]
+
+
+@pytest.mark.parametrize("device", cpu_and_gpu())
+def test_correctness_uniform_temporal_subsample(device):
+    video = torch.arange(10, device=device)[:, None, None, None].expand(-1, 3, 8, 8)
+    out_video = F.uniform_temporal_subsample(video, 5)
+    assert out_video.unique().tolist() == [0, 2, 4, 6, 9]
+
+    out_video = F.uniform_temporal_subsample(video, 8)
+    assert out_video.unique().tolist() == [0, 1, 2, 3, 5, 6, 7, 9]
```

Comment:
I've left this test in for now, but it is probably no longer needed. I tried to re-introduce my former suggestion of using `torch.linspace(..., dtype=torch.int64)` instead of `torch.linspace(...).to(torch.int64)` and the reference test triggers correctly.

Reply:
If you decide to keep this test, could you add a new line to the end of the file here in case there is actually missing one? GH shows that there is none, but I'm a little confused why our linters didn't fix or trigger on that.